installer/bundle/bundle_skel.sh

```diff
@@ -287,6 +287,28 @@ pkg_upd_list() {
     fi
 }
 
+get_arch()
+{
+  if [ $(uname -m) = 'x86_64' ]; then
+    echo "x64"
+  else
+    echo "x86"
+  fi
+}
+
+compare_arch()
+{
+  #check if the user is trying to install the correct bundle (x64 vs. x86) 
+  echo "Checking host architecture ..."
+  AR=$(get_arch)
+  case $1 in
+    *"$AR".sh) ;;
+    *)         echo "You are trying to install the wrong bundle on a ${AR} bit machine"
```

Comment:
Maybe rephrase to something more passive : "Cannot install bundle $1 on an ${AR} platform."